scripts/utils/generateRequestTypes.js

```diff
@@ -127,6 +133,18 @@ export interface ${toPascalCase(name)}${body ? `<T = ${bodyGeneric}>` : ''} exte
         return type
     }
   }
+
+  function generateExport (spec) {
+    const api = Object.keys(spec)[0]
+    const name = api
+      .replace(/\.([a-z])/g, k => k[1].toUpperCase())
```

Comment:
We repeat the same operation across several `generate` utils. I'm wondering if we can normalize data once in a centralized manner.